firebase.json

```diff
@@ -1,16 +1,16 @@
 {
-    "hosting": {
-      "public": "public",
-      "ignore": [
-        "firebase.json",
-        "**/.*",
-        "**/node_modules/**"
-      ],
-      "headers": [
-        { 
-          "source":"**/*.@(js|css|html)",
-          "headers": [
-            {"key": "Cache-Control", "value": "no-cache"}] }
-      ]
-    }
-}
\ No newline at end of file
+  "hosting": {
+    "public": "public",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "rewrites": [
```

Comment:
Please change this back

- we don't want to be doing source writing as it will prevent us hosting mutliple dashboards
- The cache control headers are important for the dashboards to continue working